### PR TITLE
UI: Add conditional enterprise logo

### PR DESCRIPTION
**Pull request by johncowen**
Adds additional 'enterprise' text underneath the 'startup' logo if the
ui is built with a CONSUL_BINARY_TYPE environment variable that doesn't
equal `oss`.

Tests to be added at a build level

### CODE REVIEW

**Comment by johncowen** [comment]
Hey @mkeeler ,

Just pinging you here as I think we should do a similar test here as we do for the UI version. Basically need to look for `data-enterprise-logo` in the same index.html file. If its there it's correct for enterprise, if you check and its not it means it ok for `oss`. Just incase, it decides by looking for that CONSUL_BINARY_TYPE thing you added for me. Shout me if anything doesn't make sense.

Ta,

**Comment by mkeeler** [comment]
@johncowen Did you want to take a crack at updating the build or did you need me to do it?

**Comment by johncowen** [comment]
Cool, yeah I can do it I can append it on here, shout you in a bit I'll add you to reviewers once its done also just incase.

Ta

File: build-support/functions/20-build.sh [hunk 1]
```diff
@@ -90,10 +95,19 @@ function build_ui {
       then
          err "ERROR: UI version mismatch. Expecting: '${version}' found '${ui_vers}'"
          ret=1
-      else
+      fi
```
[the remark this answers is not in the document]
Comment (johncowen) [reply build-support/functions/20-build.sh]: 👍 went with 3 separate blocks, nicer to read and comment in/out if you need to.

**Pull request merged**